src/Survey/ItechArt.Survey.WebApp/appsettings.json

```diff
@@ -9,5 +9,25 @@
       "Microsoft.Hosting.Lifetime": "Information"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "Serilog": {
+    "Using": [],
+    "MinimumLevel": {
+      "Default": "Information",
+      "Override": {
+        "Microsoft": "Error",
+        "System": "Error"
+      }
+    },
+    "WriteToFile": {
```

Comment:
all serilog config should be in config file
try the following:
1) AppDomain.CurrentDomain.SetData()
2) Directory.SetCurrentDirectory(AppDomain.CurrentDomain.BaseDirectory);